web/src/main/java/com/bazaarvoice/emodb/web/scanner/scanstatus/ScanStatus.java

```diff
@@ -32,7 +34,9 @@
     private final List<ScanRangeStatus> _completeScanRanges;
     private final Date _startTime;
     private Date _completeTime;
-    private final Date _compactionControlTime;
+    private Date _compactionControlTime;
+
+    private long _compactionControlBufferTimeInMillis = Duration.ofMinutes(1).toMillis();
```

Comment:
This has no setter, so when serialized to JSON the value will be lost.  I think if you add a `@JsonProperty` to it it'll get set.  Otherwise just add a bean getter method.  Or, you could remove `setCompactionControlBufferTimeInMillis()` and just add to the constructor; it's not going to change after construction.